week-5-practise/js/main.js

```diff
@@ -0,0 +1,66 @@
+var btnBlue = document.querySelector("#blueBtn");
+var btnOrange = document.querySelector("#orangeBtn");
+var btnGreen = document.querySelector("#greenBtn");
+
+var bgJumbotron = document.querySelector(".primary-content .jumbotron");
+var bikeDonate = document.querySelector(".primary-content .btn-primary");
+var volunteerButton = document.querySelector(".primary-content .btn-secondary");
+
+btnBlue.addEventListener("click", function() {
+  bgJumbotron.style.backgroundColor = "#588fbd";
+  bikeDonate.style.backgroundColor = "#ffa500";
+  volunteerButton.style.backgroundColor = "black";
+  volunteerButton.style.color = "white";
+});
+
+btnOrange.addEventListener("click", function() {
+  bgJumbotron.style.backgroundColor = "#f0ad4e";
+  bikeDonate.style.backgroundColor = "#5751fd";
+  volunteerButton.style.backgroundColor = "#31b0d5";
+  volunteerButton.style.color = "white";
+});
+
+btnGreen.addEventListener("click", function() {
+  bgJumbotron.style.backgroundColor = "#87ca8a";
+  bikeDonate.style.backgroundColor = "black";
+  volunteerButton.style.backgroundColor = "#8c9c08";
+});
+
+var btnSubmit = document.querySelector("form button");
+var formName = document.querySelector("#example-text-input");
+var formComment = document.querySelector("#exampleTextarea");
+var formMail = document.querySelector("#exampleInputEmail1");
+
+var allFields = document.querySelectorAll("form .form-control");
+
+btnSubmit.addEventListener("click", function() {
```

Comment:
You might like to try listening for a different event, because sometimes you don't actually need to click to get a form to submit - you can hit the Enter key for example. Try looking up the `submit` event (hint: this event is triggered by the form, not the button).